zone/questmgr.cpp

```diff
@@ -738,6 +738,66 @@ bool QuestManager::ispausedtimer(const char *timer_name) {
 	return false;
 }
 
+bool QuestManager::hastimer(const char *timer_name) {
+	QuestManagerCurrentQuestVars();
+
+	std::list<QuestTimer>::iterator cur = QTimerList.begin(), end;
```

Comment:
The verbosity on this code can be cut down rather significantly and use references

```
	for (auto & cur : QTimerList) {
		if (cur.mob && cur.mob == owner && cur.name == timer_name) {
			if (cur.Timer_.Enabled()) {
				return true;
			}
		}
	}
```